app/services/bolognese/writers/orcid/hyku_addons_xml_builder.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+module Bolognese
+  module Writers
+    module Orcid
+      class HykuAddonsXmlBuilder < HyraxXmlBuilder
+        # Fields guide:
+        # https://github.com/ORCID/ORCID-Source/blob/master/orcid-api-web/tutorial/works.md#work-fields
+        # rubocop:disable Metrics/MethodLength
+        def build
+          @xml[:work].title do
+            @xml[:common].title @metadata.titles.first.dig("title")
+
+            xml_subtitle
+          end
+
+          xml_short_description
+          xml_work_type
+          xml_date_published
+
+          # NOTE: A full list of external-id-type: https://pub.orcid.org/v2.1/identifiers
+          @xml[:common].send("external-ids") do
```

Comment:
Do you want to consider extracting this and the block underneath to standalone methods and call those here. 
It follows the pattern of the methods calls above where you called things like xml_date_published.

Additionally, it meant eliminate the need for rubocop override **rubocop:disable Metrics/MethodLength**

Reply:
As much as I hate changing structure to appease Rubocop, I agree that isn't bad enough for me not to do it :)